runtime-modules/forum/src/mock.rs

```diff
@@ -1074,16 +1074,6 @@ pub fn react_post_mock(
 
 /// Create default genesis config
 pub fn default_genesis_config() -> forum::GenesisConfig<Runtime> {
-    create_genesis_config(true)
-}
-
-/// Create config without data migration
-pub fn migration_not_done_config() -> forum::GenesisConfig<Runtime> {
-    create_genesis_config(false)
-}
-
-/// Create genesis config
-pub fn create_genesis_config(data_migration_done: bool) -> forum::GenesisConfig<Runtime> {
     forum::GenesisConfig::<Runtime> {
         category_by_id: vec![],
```

Comment:
I think we can also drop `category_by_id`, `thread_by_id`, `post_by_id` and `category_by_moderator` from the config now that we're dropping support for forum bootstrapping

Reply:
regarding the `_id`s I think we have in the past started id's at 1 for old pioneer .. I think it is safe to start at 0 again since new Pioneer shouldn't have any assumptions on starting id. cc: @thesan